tarantool_test.go

```diff
@@ -1,10 +1,12 @@
 package tarantool_test
```

Comment:
Maybe we also should add an example to the example files (there are already some examples for request objects).

Reply:
Added, thank you for noting.